docs/testing.md

```diff
@@ -11,6 +11,29 @@ To run the test suite locally you can use tox. By default, with no arguments, to
 tox -e 311
 ```
 
+### Test database
+
+By default, tests will run with a sqlite3 database.
+You can read more about managing the server versus sqlite3 database in `test_app/README.md`.
+
+You can also runs tests with postgres as a 2-step manual process.
+First, you need to create the postgres container (named "dab_postgres") detached, with this command.
+
+```
+make postgres
+```
+
+Now you can run tests against it, but only using the postgres-specific settings.
+
+```
+DJANGO_SETTINGS_MODULE=test_app.settings py.test test_app/tests
+```
+
+This may not work on an OS that runs docker in a VM.
+In that case, also use the `DOCKER_HOST` env var to point to wherever the VM is exposed.
+This should allow you to reproduce the results reported in the postgres database check.
+Stopping the container with `docker stop dab_postgres` will also remove the container.
```

Comment:
Need to update this, and also `test_app/README.md`. Maybe move this to that README and reference it here.